docs/contributing.rst

```diff
@@ -144,13 +145,13 @@ Running ``./test/isoltest`` again will result in a test failure:
             }
 
         Expected result:
-            DeclarationError: Identifier already declared.
+            DeclarationError: (36-52): Identifier already declared.
```

Comment:
Maybe we should mention somewhere that ``(36-52)`` are source locations? Maybe even that ``isoltest`` will try to highlight them to make it easier to verify that they are correct?